bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
I am open to any package name (ie, agree with @dguy argument). I agree that `org.apache.kafaks.streams.tools` is not optimal to use in `core`. Thus, I put it in package `kafka.tools`. About `trunk` and `0.10.0`; yes, please cherry-pick this PR to `trunk`.

Who does the final decision?